pages/resultset.py

```diff
@@ -17,21 +17,27 @@ class ResultsetPage(Base):
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
     _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
     _result_status_locator = (By.ID, 'job-details-panel')
+    _unclassified_failure_count_locator = (By.CSS_SELECTOR, '.btn-unclassified-failures')
 
     @property
     def job_result_status(self):
         return self.selenium.find_element(*self._job_result_status_locator).text
 
+    @property
+    def verify_unclassified_failure_exists(self):
+        return self.selenium.find_element(*self.unclassified_failure_count_locator).text
```

Comment:
If Treeherder added a new class to indicate unclassified failures then that would probably be the most efficient way to proceed. I just tried excluding buttons with a class containing 'unclassified' but it also returns the green, blue and grey buttons (maybe others, too). There's no way to identify failures, let alone unclassified failures. Identifying them by their colour is not ideal and really not a good practice, see http://www.w3.org/QA/Tips/goodclassnames.